guide/14-deep-learning/object-detection.ipynb

```diff
@@ -69,7 +69,7 @@
        "                        </a>\n",
```

Comment:
<p>rephrase</p><blockquote>Find the <strong>efficient</strong> learning rate</blockquote><blockquote><br></blockquote><blockquote><strong>Choosing</strong> a very small learning rate leads to very slow training of the model</blockquote><blockquote><br></blockquote><blockquote>while an extremely high rate can 'overshoot' the minima where the loss (or error rate) is lowest, and prevent the model from converging</blockquote><blockquote><br></blockquote><blockquote><code>arcgis.learn</code>&nbsp;includes fast.ai's learning rate finder, which is accessible through the model's&nbsp;<code>lr_find()</code>&nbsp;method, that can automatically select an optimum learning rate, without requiring repeated experiments.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>